PWGCF/Flow/Tasks/FlowGFWPbPb.cxx

```diff
@@ -341,6 +359,33 @@ struct FlowGFWPbPb {
       // use this cut at low multiplicities with caution
       return 0;
     }
+    float vtxz = -999;
+    if (collision.numContrib() > 1) {
+      vtxz = collision.posZ();
+      float zRes = TMath::Sqrt(collision.covZZ());
+      if (zRes > 0.25 && collision.numContrib() < 20)
+        vtxz = -999;
+    }
+
+    auto multNTracksPV = collision.multNTracksPV();
+
+    if (centrality >= 70. || centrality < 0)
+      return 0;
+    if (abs(vtxz) > cfgCutVertex)
+      return 0;
+    if (multNTracksPV < fMultPVCutLow->Eval(centrality))
+      return 0;
+    if (multNTracksPV > fMultPVCutHigh->Eval(centrality))
+      return 0;
+    if (multTrk < fMultCutLow->Eval(centrality))
+      return 0;
+    if (multTrk > fMultCutHigh->Eval(centrality))
+      return 0;
+
+    // V0A T0A 5 sigma cut
+    if (abs(collision.multFV0A() - fT0AV0AMean->Eval(collision.multFT0A())) > 5 * fT0AV0ASigma->Eval(collision.multFT0A()))
+      return 0;
+
     return 1;
```

Comment:
and `True` in here
Readability is a much for further code understanding

Reply:
Here you mean to have return 0 insted of return 1 in line 389?